### PR TITLE
xlearn 0.4.4 (new formula)

### PR DESCRIPTION
**Pull request by ankane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Hi, this adds [xLearn](https://github.com/aksnzhy/xLearn), a library for linear models, factorization machines, and field-aware factorization machines.

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]
lgtm

File: Formula/xlearn.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  def install
+    inreplace "CMakeLists.txt", "set(CMAKE_INSTALL_PREFIX \"xLearn\")", ""
+
+    mkdir "build" do
+      system "cmake", "..", "-DCMAKE_MACOSX_RPATH=TRUE", *std_cmake_args
+      system "make"
+      system "make", "install"
+
+      bin.install "xlearn_train"
```
Comment (SMillerDev) [inline Formula/xlearn.rb]: shouldn't `make install` do this?

Comment (ankane) [reply Formula/xlearn.rb]: That's what I expected as well, but it currently doesn't

**Comment by ankane** [comment]
Hey, I noticed this still has a "needs response" tag (but responded 5 weeks ago). Please let me know if the `make install` issue is a blocker and I'll communicate it to the project.

**Comment by SMillerDev** [comment]
It's not a blocker but it should be reported upstream as broken nonetheless, the response label is manual and I didn't see the notification.

**Comment by SMillerDev** [comment]
Thanks for the pull request @ankane without people like you it would be impossible to maintain homebrew to the high standards people have come to expect of the project.

**Comment by SMillerDev** [comment]
@brewtestbot test this please!

**Comment by ankane** [comment]
Great, thanks @SMillerDev! I think there may have been an issue during bottling. When installing the formula, I get:

```txt
curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "xlearn"
Download failed: https://homebrew.bintray.com/bottles/xlearn-0.4.4.mojave.bottle.tar.gz
```

**Comment by SMillerDev** [comment]
Yeah, I broke that one. https://github.com/Homebrew/homebrew-core/pull/47318 should fix it

**Comment by ankane** [comment]
Great, it's working now. Thanks for the quick fix!

**Pull request closed without merging**